packages/libs/react-ui/src/components/Stack/Stack.css.ts

```diff
@@ -0,0 +1,111 @@
+import { sprinkles, vars } from '../../styles';
+
+import { style, styleVariants } from '@vanilla-extract/css';
+
+export const container = style([
+  sprinkles({
+    display: 'flex',
+    gap: 'md',
+    flexDirection: 'row',
+    justifyContent: 'flex-start',
+    alignItems: 'flex-start',
+    flexWrap: 'nowrap',
+  }),
+  {},
+]);
+const spacingVariants: Record<string, string> = {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  '2xs': '2xs',
+  xs: 'xs',
+  sm: 'sm',
+  md: 'md',
+  lg: 'lg',
+  xl: 'xl',
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  '2xl': '2xl',
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  '3xl': '3xl',
+};
+
+export const spacingClass = styleVariants(spacingVariants, (gap) => {
+  return [
+    container,
+    {
+      gap: vars.sizes[gap as keyof typeof vars.sizes],
```

Comment:
Also, I think this mapping is a great tool, but it doesn't need to be used in all situations. For example, the variant implementations in this file are quite simple and may even be more readable and simpler if you were to explicitly write them out (I think in these case it would result in  fewer lines of code). What do you think about the difference in this situation?